fdbbackup/backup.actor.cpp

```diff
@@ -4135,6 +4171,19 @@ int main(int argc, char* argv[]) {
 
 			switch (restoreType) {
 			case RestoreType::START:
+				if ((restoreUserKeys || restoreSystemKeys) && !backupKeys.empty()) {
```

Comment:
maybe assert that both `restoreUserKeys` and `restoreSystemKeys` is set at the same time..

Reply:
I have a check above that prints and error if both are set together